app/controllers/application_controller.rb

```diff
@@ -4,6 +4,18 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :configure_permitted_parameters, if: :devise_controller?
 
+  def avg_rating
+    all_ratings = []
+    if @reviews.empty?
+      return 0.0
+    else
+      @reviews.each do |r|
+        all_ratings << r.rating.to_f
+      end
+      (all_ratings.sum / all_ratings.length).round(1)
+    end
+  end
```

Comment:
This is a method that it would make more sense to have in the `Bathroom` model so that you can call `@bathroom.average_rating`! Move it there and write a model test for it!